### ISSUE TITLE
[Bug]: Accessibility issue with Talkback

### ISSUE THREAD

**Issue by rosapanzer** [meshtastic/Meshtastic-Android#3583]
### Contact Details

bill.geiz@hotmail.com

### Checklist

- [x] I am able to reproduce the bug with the latest version.

- [x] I made sure that there are no existing **OPEN or CLOSED issues** which I could contribute my information to.

- [x] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.

- [x] This issue contains only one bug.

- [x] I have read and understood the **Contribution Guidelines**.

- [x] I agree to follow this project's Code of Conduct

### Affected app version

2.7.4

### Affected Android version

15

### Affected phone model

Asus, Pixel

### Affected node model

Wismesh Pocket

### Affected node firmware version

2.2.2020

### Steps to reproduce the bug

open the app when Talkback is active, swipe to the right until focus is at the bottom or touch the settings button, Talkback will not announce anything

### Expected behavior

Talkback should be able to announce the buttons on the bottom of the page

### Relevant log output

```shell

```

**Comment by DaneEvans** [comment]
dupe #2259
